backends-common/postgres/src/test/java/org/apache/james/backends/postgres/PostgresFixture.java

```diff
@@ -94,5 +94,6 @@ public String schema() {
         .withDatabaseName(DEFAULT_DATABASE.dbName())
         .withUsername(DEFAULT_DATABASE.dbUser())
         .withPassword(DEFAULT_DATABASE.dbPassword())
-        .withCreateContainerCmdModifier(cmd -> cmd.withName("james-postgres-test-" + UUID.randomUUID()));
+        .withCreateContainerCmdModifier(cmd -> cmd.withName("james-postgres-test-" + UUID.randomUUID()))
+        .withCommand("postgres", "-c", "max_connections=300");
```

Comment:
What if this was creating issues for some reason? might want to try remove it if I get an other failure, just for testing

Reply:
:(( just to increase number of connection allowed by postgres test container

Reply:
I guessed but I'm not sure of the impact overall... Test container seems more and more buggy lately, not sure if it's because of the lib or because we use it wrongly

Reply:
Rational being: 
- jmap integration tests are separated from event bus tests
- the only thing that I see that could change behavior in other modules is this generic container defined in postgres-backend

It's a long shot but I don't see any other candidate honestly